go.sum

```diff
@@ -4,6 +4,7 @@ github.com/cyberark/conjur-opentelemetry-tracer v0.0.1-336 h1:eyFRQDVVxjUqIO3lSG
 github.com/cyberark/conjur-opentelemetry-tracer v0.0.1-336/go.mod h1:Uq08XFlzJrRf7KbQO6mmIlwMsL1PSeYcldj34F8yGEc=
 github.com/cyberark/conjur-opentelemetry-tracer v0.0.1-394 h1:Ffl+3eKkawBUtaRC2O0rTT3KGvr/NzaO6BB7HI0fly4=
 github.com/cyberark/conjur-opentelemetry-tracer v0.0.1-394/go.mod h1:IU6D7QQezwoCi6GaKa+79ZrBNyJzFCbIAep0VrLHK6o=
+github.com/cyberark/conjur-opentelemetry-tracer v0.0.1-859/go.mod h1:knGjmz7WYYptFxOwbMTHD56oslEQrNTq2mDW9qix0fc=
```

Comment:
"go get" auto-updated the go.mod file and I undid the changes it made to that final.

Here, I forgot to remove the go.sum line generated from the auto-update made so I'll undo this line as well

Reply:
Running `go mod tidy` might fix this up for you.